Cargo.toml

```diff
@@ -26,6 +26,8 @@ bls-signatures = { version = "0.15", default-features = false, features = [
   "multicore",
   "blst-portable",
 ] } # prevent SIGINT on CI runners by using portable assembly
+blst = { version = "0.3", features = ["portable"] }
```

Comment:
I'm surprised that we need both these libraries - it looks like we're only using `blst::BLST_ERROR` - can you confirm this is necessary?

Reply:
Good catch, it should be ok to downcast to `anyhow::Error` here

Reply:
`blst` removed